backend/src/main/java/com/example/skicurort/curort/CurortMapper.java

```diff
@@ -0,0 +1,28 @@
+package com.example.skicurort.curort;
+
+import java.util.List;
+
+public class CurortMapper {
+  private CurortMapper() {}
+
+  static CurortDTO mapCurortToCurortDTO(Curort curort) {
+    return new CurortDTO(
+        curort.getCurortName(),
+        curort.getCurortAdress(),
+        curort.getCurrortEmail(),
+        curort.getCurortPhonenumber());
+  }
+
+  static Curort mapCurortDTOToCurort(CurortDTO curortDto) {
```

Comment:
nie wiem czy nie skróciłabym nazwy na po prostu mapToDTO i tak samo w kolejnym mapToEntity - ale to tylko wolna sugestja

Reply:
Oczywiście, już zmieniam